### DISCUSSION TITLE
Template slug

### DISCUSSION THREAD

**Discussion by fmenis** [mailcarrierapp/mailcarrier#4] [Q&A] [answered]
Hi @danilopolani 
can I ask why the template slug cannot be edited?
It could be useful, especially since you now need to delete the template if the slug is no longer what you want.

**Comment by danilopolani** [comment, marked as answer]
Hey,
the slug is not editable for security reasons, for example changing the slug when your template is in production could lead to undesired results.

If you need to edit it, you can rename it in your database (table `templates.slug`) or as you said delete and recreate the template.  
Please note that editing your template making the slug "no longer what you want" instead of creating a new one **should be avoided**, since templates are related also to logs and you definitely do not want to have the "Welcome" email in your logs showing the "Reset password" template 😄 

In the future (no ETA) there will be a "user permission" feature to also give more sense to the "Locked" flag on templates and layouts, but until then I don't think the slug will be editable.

**Reply by fmenis** [reply]
Hi Dani,
Your're right, for security reasons it's better that the slug can't be update!
Thanks for the explanation
